tools/appimage.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+# The MIT License (MIT)
+# Copyright (c) 2016 Simon Peter
+
```

Comment:
Add something to the effect of the following replacing the placeholders with the actual values.  Please be sure to mention that the script itself and functions.sh.

```
This code is based on <description/name of project>,  <insert copyright notice>
<location of original code>
This license of this code is an MIT License, see <url to license>
```